code/game/dna/dna_modifier.dm

```diff
@@ -114,7 +114,8 @@
 
 	if(!occupant)
 		for(var/mob/M in src)//Failsafe so you can get mobs out
-			M.forceMove(get_turf(src))
+			if(!(M.gcDestroyed))
```

Comment:
Everyone knows more parentheses means more safety

<sub>--An actual comment in my old Steambot's code</sub>